src/components/MultiCombobox/MultiCombobox.tsx

```diff
@@ -36,7 +37,12 @@ export type MultiComboboxProps<T> = {
   itemToString?: (item: T) => string;
 
   /**
-   * A function that accepts an item as a parameteer and returns `true` if the item should be
+   * A function used to group the Menu Items.
+   */
+  itemToGroup?: (item: T) => string;
```

Comment:
Similar to the above

```suggestion
  itemToGroup?: (item: T) => string | null;
```